server/handler/frontend.go

```diff
@@ -100,3 +106,11 @@ func getRequires(result *common.Result, githubURL string) map[string][]Membershi
 	}
 	return membershipInfo
 }
+
+func getPermissions(result *common.Result) []string {
+	permStrings := make([]string, len(result.Requires.Permissions))
+	for _, perm := range result.Requires.Permissions {
```

Comment:
I think you'll want to use the new `GetPermissions` function I added in #426 (once it merges) here to make sure we're including all permissions and there are no duplicates.

Reply:
Fixed, thank you!

Reply:
Oh I totally missed the "once it merges" part. Failing now, but will wait until #426 is merged.